quesma/quesma/schema_array_transformer.go

```diff
@@ -100,3 +100,87 @@ func NewArrayTypeVisitor(resolver arrayTypeResolver) model.ExprVisitor {
 	}
 	return visitor
 }
+
+func checkIfGoupingByArrayColumn(stm model.SelectCommand, resolver arrayTypeResolver) bool {
```

Comment:
what `stm` stands for?

Reply:
`stm` == `statement`

I'll change to `selectCommand`.